templates/linuxbrew.yaml

```
@@ -0,0 +1,52 @@
+# Homebrew on Linux.
+#
+# This template does not use macOS as the guest OS, because it seems hard to automate
+# provisioning macOS instances (accept EULA, skip enabling iCloud, set up SSH...).
+#
+# To run Homebrew on macOS with a sandbox user account, Alcoholless can be used instead.
+# The `alclessctl` CLI of Alcoholless is designed to be similar to `limactl`:
+# https://github.com/AkihiroSuda/alcless
+#
+# ```
+# alclessctl create default
+# cd ~/SOME_DIRECTORY
+# alcless brew install xz
+# alcless xz SOME_FILE
+# ```
+#
+# Eventually, Alcoholless may be consolidated into Lima.
+
+minimumLimaVersion: 1.1.0
+
+base:
+- template://_images/ubuntu-24.10
```

Should this be "ubuntu-latest", which we just call "ubuntu"?

```suggestion
- template://_images/ubuntu
```

---

This was intended to be `ubuntu-lts`